File: src/Test/FibFFICheatcode.php
<?php
$exports['runFibFFICheatcode'] = function($n) use (&$exports) {
    if ($n === 0) return 0;
    if ($n === 1) return 1;
    $f = $exports['runFibFFICheatcode'];
    return $f($n - 1) + $f($n - 2);
};
return $exports;

File: src/Test/FibFFICheatcode.go
package Test_FibFFICheatcode


func fib_cheatcode(n int) int {
	if n == 0 {
		return 0
	}
	if n == 1 {
		return 1
	}
	return fib_cheatcode(n-1) + fib_cheatcode(n-2)
}

func RunFibFFICheatcode(limit float64) float64 {
	dummy := int(limit)
	return float64(fib_cheatcode(dummy))
}
